vaa-app-shared/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "private": true,
-  "name": "vaa-shared",
+  "name": "vaa-app-shared",
   "version": "1.0.0",
   "scripts": {
     "package:cjs": "mkdir -p ./build/cjs/ && echo '{ \"type\": \"commonjs\" }' > ./build/cjs/package.json",
     "package:esm": "mkdir -p ./build/esm/ && echo '{ \"type\": \"module\" }' > ./build/esm/packagec.json",
     "build:cjs": "yarn package:cjs && yarn tsc --build tsconfig.cjs.json",
     "build:esm": "yarn package:esm && yarn tsc --build tsconfig.esm.json",
     "build": "yarn build:cjs && yarn build:esm",
-    "test": "vitest run tests",
```

Comment:
I'm moving towards housing unit test files next to the affected source files for easier checking of test coverage and updates to tests when the source changes.